### PR TITLE
Fix some mob lua's

### PR DESCRIPTION
**Pull request by ymkc**
- moved left over mobs  from killer to isKiller
- updateEnmity -> updateClaim (verified on yt)
- Cerberus initial pop was set too high
- Bastok 9-1 Mission mob popped with updateClaim

### CODE REVIEW

File: scripts/zones/VeLugannon_Palace/mobs/Detector.lua [hunk 1]
```diff
@@ -77,18 +77,7 @@ end;
...
-
-    local Detector = mob:getID();
-    local Caretaker = mob:getID() + 1;
-
-    GetMobByID(Detector):resetLocalVars();
-
-    if (GetMobAction(Caretaker) ~= 0) then
-        DespawnMob(Caretaker);
-    end
-
```
Comment (takhlaq) [inline scripts/zones/VeLugannon_Palace/mobs/Detector.lua]: why was this removed? (no idea what's supposed to happen in retail but assuming whatever was here before was retail-ish)

Comment (ymkc) [inline scripts/zones/VeLugannon_Palace/mobs/Detector.lua]: the same code was in there twice, once under onMobDeath and once under onMobDespawn

Comment (takhlaq) [inline scripts/zones/VeLugannon_Palace/mobs/Detector.lua]: derp didn't see that lol

**Comment by TeoTwawki** [comment]
Thank you for taking care of these.
